ui/src/app.jsx

```diff
@@ -113,15 +114,27 @@ export default React.createClass({
   },
 
   messagePopupAuthorQuestions(query = {}) {
-    return <MessagePopup.QuestionsPage />;
+    return (
+      <QuestionsLoader>
```

Comment:
Oh also let's remove any logic at all from the `app` page so that it only has responsibility for deciding what page to render.  Since all the routes have to go through the `app` page it will continue to grow for new pages we add, so keeping everything else out can help make it more manageable.

So these can move into `NewQuestionPage` etc.